golle.sublime-project

```diff
@@ -1,8 +1,16 @@
 {
```

Comment:
It's easier if each developer keeps his own project configuration, because some of us use weird settings that do things like saving draft copies of our own work.